adagrams/game.py

```diff
@@ -1,11 +1,126 @@
+import random
+from collections import Counter
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+    hand = random.sample(list(LETTER_POOL.keys()), 10)
+    return hand
 
+# def uses_available_letters(word, letter_bank):
+#     # Convert word to uppercase
+#     word = word.upper()
+#     # Create a counter of the letter_bank
+#     bank_counts = Counter(letter_bank)
+#     # Iterate over each letter in the word
+#     for letter in word:
+#         # If the letter is not in the bank_counts, or if the count is 0, return False
+#         if letter not in letter_bank or bank_counts[letter] == 0:
+#             return False
+#         # Otherwise, decrement the count of the letter in bank_counts
+#         bank_counts[letter] -= 1
+#     # If we make it through the loop without returning False, return True
+#     return True
 def uses_available_letters(word, letter_bank):
-    pass
+    word = word.upper()
+    word_counts = Counter(word)
```

Comment:
Great use of a built-in! For `Counter` as with the `sample` function above, please consider including a comment about how you understand the function to work, and how you researched it.